models/google_play__finance_report.sql

```diff
@@ -17,7 +17,7 @@ product_info as (
     select *
     from {{ ref('int_google_play__latest_product_info') }}
 
--- there's honestly quite a bit in here since we only need to do backfilling stuff if there is indeed a full outer join
+-- there's honestly quite a bit in here since we only need to do backfilling stuff if there is indeed a full outer join 
```

Comment:
Not sure I am following this comment. Is this entirely necessary?

Reply:
ah probably not. i meant that there's a lot of code encapsulated in the following `if` bloc, which isn't really necessary